File: app/src/main/java/com/aslansari/hypocoin/repository/model/Account.java
package com.aslansari.hypocoin.repository.model;

import androidx.annotation.NonNull;
import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

import org.jetbrains.annotations.NotNull;

@Entity
public class Account {
    @PrimaryKey
    @NonNull
    public final String id;

    @ColumnInfo(name = "balance")
    private long balance;

    public Account(@NotNull String id) {
        this.id = id;
    }

    public long getBalance() {
        return balance;
    }

    public void setBalance(long balance) {
        this.balance = balance;
    }
}

File: app/src/main/java/com/aslansari/hypocoin/repository/model/Account.kt
package com.aslansari.hypocoin.repository.model

import androidx.room.PrimaryKey
import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.Ignore

@Entity
data class Account(
    @field:PrimaryKey val id: String,
    val email: String,
) {
    @ColumnInfo(name = "balance")
    var balance: Long = 0

    @Ignore var passwordPlaintext: String = ""
}